src/module.zig

```diff
@@ -149,6 +159,7 @@ pub fn Objects(comptime T: type) type {
             const dead = &objs.internal.dead;
             const generation = &objs.internal.generation;
             const recycling_bin = &objs.internal.recycling_bin;
+            const _updated = &objs.internal.updated;
```

Comment:
nit: prefer avoiding underscores, we can rename one to `updated_fields` instead?